crates/project-model/src/project_json.rs

```diff
@@ -362,6 +364,8 @@ struct CrateData {
     repository: Option<String>,
     #[serde(default)]
     build: Option<BuildData>,
+    #[serde(default)]
+    cwd: Option<Utf8PathBuf>,
```

Comment:
```suggestion
    proc_macro_cwd: Option<Utf8PathBuf>,
```
@davidbarsky for input here, unsure if we need to model this in the project json payload or not but I don't think we can derive the correct one from the data.

Reply:
I *think* for rust-project.json payloads, we should be conservative and mimic what Cargo does, but I'm not 100% sure.